config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -14,6 +14,10 @@ presubmits:
     always_run: true
     path_alias: k8s.io/kubernetes
     spec:
+      # unit tests have no business requiring root or doing privileged operations
+      securityContext:
+        runAsUser: 2000
```

Comment:
Does that user exist? Tests tend to do stuff in $HOME which if inexistent results in writing to / which wont work

Reply:
It doesn't, and they don't, unit tests should write to a tempdir and not the real $HOME.

We already had a test version of the job (`pull-kubernetes-unit-experimental`) that passes with this config.